test/Microsoft.ML.Benchmarks.Tests/BenchmarksTest.cs

```diff
@@ -39,14 +42,25 @@ public class BenchmarksTest
 
         private ITestOutputHelper Output { get; }
 
-#if DEBUG
-        [Fact(Skip = SkipTheDebug)]
+#if DEBUG || NET462
+        [ConditionalTheory(Skip = SkipTheDebugAndFullFramework)]
 #else
-        [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))]
+        [ConditionalTheory(typeof(Environment), nameof(Environment.Is64BitProcess))]
 #endif
-        public void BenchmarksProjectIsNotBroken()
+        // [InlineData(typeof(RankingTrain))]
+        // [InlineData(typeof(RankingTest))]
+        // [InlineData(typeof(MultiClassClassificationTrain))]
+        // [InlineData(typeof(MultiClassClassificationTest))]
```

Comment:
dead code (commented lines): you should either add comments explaining why those benchmarks are not executed or remove the code